### PR TITLE
Startup modal polish

### PR DESCRIPTION
**Pull request by cosmicpotato137**
## Describe your changes

Added separate qr code step, fixed alignment issues from last pr.

## Issue ticket number and link
Closes #813 

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [x] I have tested on a mobile device
- [ ] It has been deployed to people-test.sphinx.chat and tested by others

### CODE REVIEW

File: frontend/app/src/people/utils/StartUpModal.tsx [hunk 1]
```diff
@@ -76,24 +76,15 @@ const StartUpModal = ({ closeModal, dataObject, buttonColor }: StartUpModalProps
...
-        }
-      </ModalContainer>
-      <ButtonContainer>
-        <IconButton
-          text={'Back'}
-          width={210}
-          height={48}
-          buttonType={'text'}
-          style={{ color: '#83878b', marginTop: '0px', textDecoration: 'none' }}
+        {!connection_string ? (
```
Comment (cosmicpotato137) [inline frontend/app/src/people/utils/StartUpModal.tsx:79]: Not sure how this got corrupted, fix incoming

**Comment by cosmicpotato137** [comment]
Not sure how to fix the Prettier check...

**Comment by kevkevinpal** [comment]
> Not sure how to fix the Prettier check...

if you run `yarn run prettier` in `sphinx-tribes/frontend/app` it should auto format the project and then just commit the changes

**Comment by kevkevinpal** [comment]
also just as a future heads up screenshots really help out the review process to help any reviews understand what exactly is being added/modified

**Comment by kevkevinpal** [comment]
prettier failure addressed in [3e21f6b](https://github.com/stakwork/sphinx-tribes/pull/821/commits/3e21f6b613005c14916d9002fb67bc8a876997e7)

**Pull request merged**